test/unit/components/sprite-selector-item.test.jsx

```diff
@@ -1,11 +1,11 @@
 /* eslint-env jest */
 import React from 'react'; // eslint-disable-line no-unused-vars
-import {shallow} from 'enzyme';
+import {mountWithIntl, shallowWithIntl, componentWithIntl} from '../../helpers/intl-helpers';
 // eslint-disable-next-line no-unused-vars
 import SpriteSelectorItemComponent from '../../../src/components/sprite-selector-item/sprite-selector-item';
 import CostumeCanvas from '../../../src/components/costume-canvas/costume-canvas';
 import CloseButton from '../../../src/components/close-button/close-button'; // eslint-disable-line no-unused-vars
-import renderer from 'react-test-renderer';
+import {MenuItem} from '../../../src/components/context-menu/context-menu'; // eslint-disable-line no-unused-vars
```

Comment:
Woops, that is the danger of disabling eslint. I used in a previous iteration of the test and didn't remove it. Fixed.